subgraphs/_reference_/src/prices/common/types.ts

```diff
@@ -35,22 +35,27 @@ export class CustomPriceType {
   private _usdPrice: Wrapped<BigDecimal>;
   private _decimals: Wrapped<i32>;
   private _oracleType: string;
+  private _counterLiquidity: Wrapped<BigDecimal>;
```

Comment:
Why not just call it `_liquidity` I am slightly confused why it is called `_counterLiquidity`